src/platform_impl/windows/window_state.rs

```diff
@@ -42,9 +48,100 @@ pub struct SavedWindow {
     pub placement: winuser::WINDOWPLACEMENT,
 }
 
-#[derive(Clone)]
+pub struct RgbaHandle {
+    pub hcursor: AtomicPtr<HICON__>,
```

Comment:
It would be good to reuse the `RaiiIcon` struct here imo, mirroring `WinIcon`

Reply:
Actually `HICON__` is a raw pointer that can't be sent, I can't use `RaiiIcon` as is :
```
error[E0277]: `*mut HICON__` cannot be sent between threads safely
   --> src/platform_impl/windows/window.rs:556:30
    |
556 |         self.thread_executor.execute_in_thread(move || {
    |                              ^^^^^^^^^^^^^^^^^ `*mut HICON__` cannot be sent between threads safely
    |
    = help: within `WindowState`, the trait `Send` is not implemented for `*mut HICON__`
```

Reply:
My remark is outdated because of the switch to windows_sys that uses integer instead of raw pointers. I'll try to update it.